e2e/pages/createClaim/uploadParticularsOfClaimQuestion.page.js

```diff
@@ -0,0 +1,27 @@
+const { I } = inject();
+
+module.exports = {
+
+  fields: {
+    uploadQuestion: {
+      id: '#uploadParticularsOfClaim',
+      options: {
+        yes: 'Yes',
+        no: 'No'
+      }
+    },
+  },
+
+  async chooseIfUploadParticularsOfClaim(uploadParticularsOfClaimOption) {
+    // eslint-disable-next-line no-prototype-builtins
+    if (!this.fields.uploadQuestion.options.hasOwnProperty(uploadParticularsOfClaimOption)) {
+      throw new Error(`Respondent represented option: ${uploadParticularsOfClaimOption} does not exist`);
```

Comment:
copy and paste, please update. Although not sure this level of throwing an exception etc is required. Just have a choose yes / choose no function perhaps? Only the yes function will probably be used in e2e